product/components/ProductDrawer.tsx

```diff
@@ -22,9 +22,10 @@ interface Props {
 
 const ProductDrawer: React.FC<Props> = ({isOpen, defaultValues, onClose, onSubmit}) => {
   const isNew = Boolean(!defaultValues?.id);
+  const closeOnOverlayClick = false;
 
   return (
-    <Drawer id="product" isOpen={isOpen} placement="right" size="md" onClose={onClose}>
+    <Drawer id="product" isOpen={isOpen} placement="right" size="md" onClose={onClose} closeOnOverlayClick={closeOnOverlayClick}>
```

Comment:
```suggestion
    <Drawer id="product" isOpen={isOpen} placement="right" size="md" onClose={onClose} closeOnOverlayClick={false}>
```

Lo podemos poner inline?

Reply:
Claro, ayer no me di cuenta de poner el `false` entre {} y rompía; por eso creé la constante. Mucho mejor así.